# HG changeset patch
# User Pulkit Goyal <pulkit@yandex-team.ru>
# Date 1542717426 -10800
# Node ID 04a24a13960ff1545a49707541e748de191304dd
# Parent  74c5fd9c3e762d82ee8957acd384d00dc91fc2cb
util: add visitdir() and visitchildrenset() to PrefixMatch

This is done to be compatible with treemanifest repositories.

diff --git a/hgsubversion/util.py b/hgsubversion/util.py
--- a/hgsubversion/util.py
+++ b/hgsubversion/util.py
@@ -249,6 +249,12 @@
     def prefix(self):
         return False
 
+    def visitdir(self, dir):
+        return True
+
+    def visitchildrenset(self, dir):
+        return 'this'
+
 def outgoing_revisions(repo, reverse_map, sourcerev):
     """Given a repo and an hg_editor, determines outgoing revisions for the
     current working copy state.
